File: planik-android/app/src/main/java/pl/planik/domain/model/DayEntry.kt
package pl.planik.domain.model

import java.time.DayOfWeek
import java.time.OffsetDateTime

data class DayEntry(
  val id: Int,
  val dayOfWeek: DayOfWeek,
  val title: String,
  val start: OffsetDateTime,
  val end: OffsetDateTime,
  val pauseMinutes: Int,
)

File: planik-flutter/planik/lib/domain/model/day_entry.dart
import 'package:freezed_annotation/freezed_annotation.dart';

part 'day_entry.freezed.dart';

@freezed
class DayEntry with _$DayEntry {
  const factory DayEntry({
    required int id,
    required int dayOfWeek,
    required String title,
    required DateTime start,
    required DateTime end,
    required int pauseMinutes,
  }) = _DayEntry;
}
